### PR TITLE
fix: issues with main menu

### PR DESCRIPTION
**Pull request by damianmuti**
Updated logic to emulate Doppler App menu.
*NOTE*: I did not do any improvements in terms of code, just focused on fixing the issue.

**DEMO:** https://cdn.fromdoppler.com/doppler-ui-library/build248/header.html

### CODE REVIEW

File: assets/js/main.js [hunk 1]
```diff
@@ -1,15 +1,19 @@
-$("body").on({
+$('.menu-main>li').on({
```
Comment (andresmoschini) [inline assets/js/main.js]: Mmm, este es un cambio peligroso, si podés a la tarde lo charlamos.

File: assets/js/main.js [hunk 2]
```diff
@@ -1,15 +1,19 @@
-$("body").on({
+$('body').on({
   mouseenter: function() {
     var submenu = $(this).find(".sub-menu");
 
+    if(!$(this).hasClass('submenu-item'))
+      $('.header-main').removeClass('header-open');
```
Comment (andresmoschini) [inline assets/js/main.js]: Very minor, could you use always curly braces like in other places?

File: assets/scss/templates/_header.scss [hunk 3]
```diff
@@ -13,11 +13,14 @@
...
+  min-height: 70px;
   flex-direction: column;
+  transition: all 0.15s ease;
 
-  &.header-open,
-  &.header-is-active {
-    height: 105px;
+  &.header-open {
+    min-height: 105px;
+    overflow: visible;
```
Comment (andresmoschini) [inline assets/scss/templates/_header.scss]: _header-is-active_ is not necessary anymore?

File: assets/templates/header.html [hunk 4]
```diff
@@ -46,7 +46,7 @@
     <li class="nav-item"><a href="forgot-password.html" class="nav-link">Forgot</a></li>
   </ul><!-- nav -->
 </section><!-- section-module -->
-    <header class="header-main header-is-active">
+    <header class="header-main header-open">
```
Comment (andresmoschini) [inline assets/templates/header.html]: Does it requires changes in webapp?

**Comment by andresmoschini** [comment]
Buen trabajo @damianmuti!

No pude seguir mucho la lógica, pero lo probé y se ve muy bien.

Si querés en la reunión de hoy de las 14 lo probamos con la webapp.

**Comment by cbernat** [comment]
It's looking very nice! 👍

File: assets/js/main.js (hunk 1 above)
Comment (damianmuti) [reply assets/js/main.js]: Fixed

File: assets/js/main.js (hunk 2 above)
Comment (damianmuti) [reply assets/js/main.js]: Sure!

File: assets/scss/templates/_header.scss (hunk 3 above)
Comment (damianmuti) [reply assets/scss/templates/_header.scss]: Exactly. That class is not necessary any more

File: assets/templates/header.html (hunk 4 above)
Comment (damianmuti) [reply assets/templates/header.html]: Yeah, and I'lll push a PR on the webapp with the changes as well.

**Comment by andresmoschini** [comment]
Could you take a look to this issue?

image

**Comment by andresmoschini** [comment]
@GusBaamonde @mvillaseco @fcoronelmakingsense could you take a look?

File: assets/scss/templates/_header.scss [hunk 5]
```diff
@@ -90,7 +93,8 @@
 
   li {
     margin-right: 25px;
-    padding: 10px 10px 10px 0;
+    padding: 0 10px 0 0;
```
Comment (fcoronelmakingsense) [inline assets/scss/templates/_header.scss]: is it easier put `padding-right: 10px` not?

Comment (andresmoschini) [reply assets/scss/templates/_header.scss]: Maybe @damianmuti wants to ensure 0 padding.

**Comment by andresmoschini** [comment]
:shipit:

**Pull request merged**